content/locations.json

```diff
@@ -0,0 +1 @@
20648683": "West Passage", "3722165744": "WaterGate", "3733413405": "Dining Hall", "3759497513": "Lockup", "3759590144": "Tower", "3761613288": "Guards Chambers", "3765504630": "West Passage", "3776459809": "Tower", "3783789786": "East Hallway", "3792015483": "Lockup", "3798558707": "Docks", "3803281416": "Observation Room Balcony", "3809238677": "East Passage", "3809555468": "Library", "3810133612": "Library", "3810322508": "Library", "3815277239": "Library", "3822768126": "Guards Chambers", "3825158129": "Lockup", "3835931569": "Guards Chambers", "3844696999": "Docks", "3870691984": "East Hallway", "3870929515": "West Passage", "3871643211": "East Hallway", "3873160272": "East Passage", "3884050780": "Tower", "3885473021": "Docks", "3895075760": "Lockup", "3898697910": "Dining Hall", "3903614698": "Docks", "3921413042": "Observation Room", "3928239822": "Library", "3930597103": "Observation Room Balcony", "3945197177": "Lockup", "3947693507": "East Hallway", "3948053634": "Lockup", "3952390696": "Zeldas Study", "3958292630": "Dining Hall", "3976257686": "Dining Hall", "3980714053": "Tower", "3981612516": "Docks", "3986555859": "Tower", "4017668815": "Dining Hall", "4018682418": "East Hallway", "4019115026": "East Hallway", "4019322377": "East Passage", "4019654697": "West Passage", "4024662738": "West Passage", "4025103369": "East Hallway", "4025584777": "East Hallway", "4025818642": "West Passage", "4026187497": "East Hallway", "4026290377": "East Hallway", "4029714219": "East Passage", "4037245333": "Lockup", "4040768850": "Lockup", "4042980639": "Tower", "4048077713": "Guards Chambers", "4056329029": "Library", "4056993669": "Library", "4057342373": "Library", "4057726437": "Library", "4065492964": "Docks", "4080067016": "Guards Chambers", "4086360230": "Zeldas Room", "4094436121": "West Passage", "4094888194": "WaterGate", "4100034009": "WaterGate", "4100267842": "East Passage", "4100636601": "East Hallway", "4101208962": "West Passage", "4101346722": "East Passage", "4101523938": "West Passage", "4101649753": "East Hallway", "4102312882": "Observation Room", "4103218707": "Tower", "4111627184": "Lockup", "4125016520": "Guards Chambers", "4130449374": "Guards Chambers", "4131038440": "Castle Tower", "4169217372": "Library", "4169505790": "East Hallway", "4169795388": "Library", "4170099102": "West Main Corridor", "4170878428": "Library", "4187193099": "Lockup", "4198495238": "Tower", "4203981551": "Dining Hall", "4221888462": "Stairs", "4231401856": "Docks", "4236085999": "Dining Hall", "4237591441": "Guards Chambers", "4237607412": "West Passage", "4247023483": "West Passage", "4247131072": "West Main Corridor", "4247368507": "East Passage", "4247811840": "East Hallway", "4248213792": "West Main Corridor", "4248386011": "West Passage", "4248520032": "East Hallway", "4248677184": "West Main Corridor", "4261783734": "Dining Hall", "4265695915": "Observation Room", "4271435072": "Scantum", "4274460657": "East Hallway", "4292311721": "Lockup"}
```

Comment:
The keys should be ints, not strings.

There are a couple of typos that should be fixed:

* Scantum -> Sanctum
* Guards Chambers -> Guards' Chambers
* Zeldas Room -> Zelda's Room
* What does WaterGate refer to?

Reply:
I was unsure of what to call the WaterGate location, but it is know as the Copium Route in any%.  It is also the gate on the front of the castle path leading up to the left (west) that can take you to the guards chambers.

Reply:
Copium gate? Hmmmmm I don't really know what to call it either... Just thought [Watergate](https://en.wikipedia.org/wiki/Watergate_scandal) sounded a bit funny.

Maybe we should ask about it on the speedrunning server